chromium/.eslintrc.json

```diff
@@ -1,16 +1,16 @@
 {
   "env": {
     "browser": true,
-    "commonjs": true,
     "es6": true,
-    "node": true,
     "webextensions": true
   },
   "parserOptions": {
     "ecmaVersion": 2017,
     "sourceType": "script"
   },
   "rules": {
-    "indent": [2, 2]
+    "indent": ["error", 2],
+    "no-unused-vars": "error",
+    "no-undef": "error"
   }
```

Comment:
Do you think it would be easier to move some of the globals into the eslintrc like:
```javascript
  "globals": {
    "DBUG"...
```
I'm not sure what the exact format is supposed to looks like, but I think eslint supports it. [docs here](https://eslint.org/docs/user-guide/configuring).

Reply:
It will cause an error in `util.js` since we will be redefining the already existing global.